syntax = "proto3";

service pyLLMService {
  rpc GenerateResponse(LLMRequest) returns (stream LLMResponse);
}

message LLMRequest {
  string birth_date = 1;  // 出生日期
  string birth_time = 2;  // 出生时间
}

message LLMResponse {
  string chunk = 1;  // 每次返回的流式内容
}
